modules/databricks-loader/src/main/resources/application.conf

```diff
@@ -5,6 +5,8 @@
         "loadAuthMethod": {
             "type": "NoCreds"
             "roleSessionName": "rdb_loader"
-        }
+        },
+       "analyzeEventsSchedule": "0 0 0 ? * *",
+       "analyzeManifestSchedule": "0 0 5 ? * *"
```

Comment:
Just sharing some thoughts.... I'm not saying to change anything (yet!)

Did you consider making these part of the `schedules` section of the config, as it already exists already, and seems similar? [Here](https://github.com/snowplow/snowplow-rdb-loader/blob/4.2.1/config/databricks.config.reference.hocon#L61).

Why did you choose `analyze*` instead of `optimize*`?  I mean, the databricks name for what we're doing is "optimize".

Sorry for nitpicking on this file.  I consider the configuration file to be the most important file in a code review, because it's the one part of the application that our users interact with directly.

Reply:
I think that is a good idea, as it would be adding redshift statements a bit easier.

Reply:
Sorry to nitpick again...  Now that you've put it in the `schedules` section, doesn't it make sense to call the key `optimizeEvents` instead of `optimizeEventsSchedule`?  Otherwise it's repeating the word "schedule".